dbt/models/iasworld/schema/iasworld.aasysjur.yml

```diff
@@ -1,13 +1,16 @@
 sources:
   - name: iasworld
-    loaded_at_field: date_parse(wen, '%Y-%m-%d %H:%i:%s.0')
+    loaded_at_field: date_parse(loaded_at, '%Y-%m-%d %H:%i:%S.%f')
     tags:
       - load_auto
       - test_qc_iasworld
 
     tables:
       - name: aasysjur
         description: '{{ doc("table_aasysjur") }}'
+        freshness:
+          warn_after: {count: 192, period: hour} # 8 days
+          error_after: {count: 360, period: hour} # 15 days
```

Comment:
Not all tables get daily updates; some get pulled every Saturday. We expect the latter to be loaded within the last 8 days.